src/System.Private.CoreLib/shared/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -618,21 +618,21 @@ public static void ThrowExceptionForHR(int errorCode)
         {
             if (errorCode < 0)
             {
-                throw GetExceptionForHR(errorCode, IntPtr.Zero);
+                throw GetExceptionForHR(errorCode, IntPtr.Zero)!;
             }
         }
 
         public static void ThrowExceptionForHR(int errorCode, IntPtr errorInfo)
         {
             if (errorCode < 0)
             {
-                throw GetExceptionForHR(errorCode, errorInfo);
+                throw GetExceptionForHR(errorCode, errorInfo)!;
```

Comment:
This might be an interesting scenario in that we know the result isn't null here; since it is only `null` for positive errorCode.

Reply:
> This might be an interesting scenario in that we know the result isn't null here; since it is only null for positive errorCode.

Consider adding a suggested attribute to https://github.com/dotnet/roslyn/issues/26761.

Reply:
Done.